ubuntudesign/vanilla-framework
# Lists overhaul

## Done

Reworked/added the following lists and items:
 - .list
 - .list__item--icon
 - .list--divided
 - .inline-list
 - .inline-list__item--middot

## QA

Build vanillaframework.io using this sass and check the rendering and code examples on /patterns/lists/

## Details

Partial fix for the issues raised here (with some agreed changes from the original naming suggestions): https://github.com/ubuntudesign/vanilla-framework/issues/349
